iodata/test/test_molekel.py

```diff
@@ -157,3 +163,10 @@ def test_load_mkl_h2_huge_threshold():
         warnings.simplefilter("error")
         # The threshold is set very high, which skip a correction for ORCA.
         load_one(str(fn_molekel), norm_threshold=1e4)
+
+
+def test_generalized():
+    # The Molden format does not support generalized MOs
+    data = create_generalized()
+    with pytest.raises(PrepareDumpError):
+        dump_one(data, "generlized.mkl")
```

Comment:
`generalized.mkl` unless there was a reason to keep it to 10 characters or less.

Reply:
fixed